src/utils/get-in.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Gets a deeply-nested property value from an object, given a 'path'
+ * of property names.
+ */
+export function getIn(
+  v: Object,
+  pathElems: string[]): any {
+    const [ firstElem, ...restElems] = pathElems;
+    if (!v) {
+      return v;
+    }
+
+    if (undefined === v[firstElem]) {
```

Comment:
is there any reason of why are you not doing
```
if (undefined === v[firstElem]) {
  return v[firstElem];
}
```
and you could delete from line 9 - 11 ?

Because if you have a case where one of the properties resolve to null. Your code wouldn't break but it would only return in the next recursive call.